_sass/bootstrap/_scaffolding.scss

```diff
@@ -20,7 +20,7 @@
 // Body reset
 
 html {
-  font-size: 10px;
+  font-size: 13px;
```

Comment:
No. We set an overriding `<body>` style in `main.css`. This only affects rem unites (which I used in REPL).